internal/pod_models.go

```diff
@@ -973,23 +992,23 @@ func GetBackupDeployment(backup *fdbv1beta2.FoundationDBBackup) (*appsv1.Deploym
 	return deployment, nil
 }
 
-// GetStorageServersPerPodForPod returns the value of STORAGE_SERVERS_PER_POD from the sidecar or 1
-func GetStorageServersPerPodForPod(pod *corev1.Pod) (int, error) {
+// GetServersPerPodForPod returns the count of servers per Pod based on the processClass from the sidecar or 1
+func GetServersPerPodForPod(pod *corev1.Pod, pClass fdbv1beta2.ProcessClass) (int, error) {
```

Comment:
The process class can be fetched from the Pod metadata: https://github.com/FoundationDB/fdb-kubernetes-operator/blob/main/pkg/podmanager/fdb_process_group.go#L57